### DISCUSSION TITLE
Can't delete robots txt in public folder

### DISCUSSION THREAD

**Discussion by ricick** [vercel/vercel#6637] [Help]
We added a robots.txt to our public folder of the production deployment while we were running our site on a test domain until we were ready to go public.

I tried changing the robots file to allow google but the old one is still serving. I tried deleting it, but vercel is still serving the cached file.

Is there any way to clear this from cache?

**Comment by samcx** [comment]
Hi there,

There is no way to specifically purge a certain cache, but it should be cleared when you redeploy. Did you already send us a request at https://vercel.com/support/request?

**Reply by ricick** [reply]
Solved this. it was our mistake, we were overwriting this in a build step with next-sitemap
